src/cli/cli_app.js

```diff
@@ -429,6 +429,9 @@ class CLI {
         status: { url: this._statusUrl }
       };
     }
+
+    nodeConfig.extraConfig.mgmt = { mgmtBase: { } };
```

Comment:
Please add a flag to the CLI that will manage the management server (same as for the web server, to allow initializing the worker without it)